oep-template.rst

```diff
@@ -50,32 +54,20 @@ existing architecture or process is inadequate to address the problem that the
 OEP solves, or why adopting the best practice would significantly improve Open
 edX.
 
-OEP submissions without sufficiently clear and compelling motivation are
-unlikely to be accepted.
-
 Specification
 =============
 
 The specification describes the technical details of the Architecture, Best
-Practice, Process or Product Enhancement proposed by the OEP. If the proposal
-includes a new API, specify its syntax and semantics.
+Practice or Process proposed by the OEP. If the proposal includes a new API,
```

Comment:
I understand and generally agree with your rational for removing Product Enhancements. I do wonder though how those should be brought up and resolved. Can we spare a few words in this document to say what is out of scope for OEPs?

Reply:
@pdpinch Yes. Good idea to say where one should go instead for Product Enhancements. I've reached out to the edX Product team on what their current process is. 
FYI  @marcotuts @sstack22

Reply:
@pdpinch We, including @johnmark, are working to determine a good process for Product Enhancements (for us to use internally as well as with the open edX community).  Please stay tuned to provide input on this.